docs/docs/examples/model-inference/S3-Model-Inference/index.md

```diff
@@ -137,37 +155,23 @@ bacalhau docker run \
 -i src=s3://sagemaker-sample-files/datasets/image/MNIST/model/pytorch-training-2020-11-21-22-02-56-203/model.tar.gz,dst=/model/,opt=region=us-east-1 \
 -i git://github.com/js-ts/mnist-test.git \
 pytorch/pytorch \
- -- python /inputs/js-ts/mnist-test/inference.py --tar_gz_file_path /model/model.tar.gz --output_directory /model-pth --image_path /inputs/js-ts/mnist-test/image.png
+ -- python3 /inputs/js-ts/mnist-test/inference.py --tar_gz_file_path /model/model.tar.gz --output_directory /model-pth --image_path /inputs/js-ts/mnist-test/image.png)
 ```
+When the job is submitted Bacalhau prints out the related job id. We store that in an environment variable `JOB_ID` so that we can reuse it later on.
 
-### Structure of the command
-
----
-
-* `-w /inputs` Setting the current working directory at /inputs in the container
 
-* `-i src=s3://sagemaker-sample-files/datasets/image/MNIST/model/pytorch-training-2020-11-21-22-02-56-203/model.tar.gz,dst=/model/,opt=region=us-east-1`: Mounting the s3 bucket at the destination path provided
-  `/model/` and specifying the region where the bucket is located `opt=region=us-east-1`
+### Viewing the Output
 
-* `-i git://github.com/js-ts/mnist-test.git`: Flag to mount the source code repo from GitHub. It would mount the repo at `/inputs/js-ts/mnist-test`
-  in this case it also contains the test image.
+Use the `bacalhau logs` command to view the job output, since the script prints the result of execution to the stdout:
 
-* `pytorch/pytorch`: The name of the Docker image.
-
-* `-- python /inputs/js-ts/mnist-test/inference.py --tar_gz_file_path /model/model.tar.gz --output_directory /model-pth --image_path /inputs/js-ts/mnist-test/image.png`: The command to run inference on the model.
-
-  - `/model/model.tar.gz` is the path to the model file.
-
-  - `/model-pth` is the output directory for the model.
-
-  - `/inputs/js-ts/mnist-test/image.png` is the path to the input image.
-
-The job has been submitted and Bacalhau has printed out the related job id. We store that in an environment variable so that we can reuse it later on.
+```bash
+bacalhau logs ${JOB_ID}
 
-### Viewing the output
+Predicted class: 0
```

Comment:
Not sure what this is for?

Reply:
This is an example of the job output using the script and image presented in the article. We provide examples of output in other articles as well. Should it be removed from here? Or should we add a comment that this is an example of output to make it more clear?